package.json

```diff
@@ -11,15 +11,14 @@
   "dependencies": {
     "async": "0.9.2",
     "axios": "0.7.0",
-    "babel": "5.8.38",
-    "babel-cli": "^6.11.4",
-    "babel-core": "6.7.7",
-    "babel-eslint": "6.1.2",
-    "babel-plugin-add-module-exports": "0.1.4",
-    "babel-plugin-lodash": "2.3.0",
-    "babel-preset-es2015": "6.6.0",
-    "babel-preset-stage-0": "6.5.0",
-    "babel-register": "6.7.2",
+    "babel": "^6.5.2",
```

Comment:
We should remove all carets: http://stackoverflow.com/questions/22343224/difference-between-tilde-and-caret-in-package-json